### PR TITLE
Feat - Switch from SSM to outputs + SAM

### PR DESCRIPTION
**Pull request by g-pelletier**
Switch from SSM parameters to outputs, which is much faster.
Also switched to SAM and sentry-sdk to be more up-to-date.
Added a dummy resource because it is required in the template.

### CODE REVIEW

File: exporter/lambda/cross_region_import_replication.py [hunk 1]
```diff
@@ -5,92 +5,73 @@
 
 import boto3
 import botocore
-from raven import Client
-from raven.transport import HTTPTransport
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+
+sentry_sdk.init(integrations=[AwsLambdaIntegration()])
```
Comment (daxAKAhackerman) [inline exporter/lambda/cross_region_import_replication.py:11]: I believe this is what we use in the newer projects: 
```suggestion
sentry_sdk.init(integrations=[AwsLambdaIntegration(timeout_warning=True)])
```

File: exporter/lambda/cross_region_import_replication.py [hunk 2]
```diff
@@ -100,63 +81,49 @@ def _generate_hash(string_to_hash):
...
-    resource_count = 0
+def _generate_nested_template(cross_stack_references):
+    template = {
+        "AWSTemplateFormatVersion": "2010-09-09",
+        "Resources": {
+            "PlaceHolderParameter": {"Type": "AWS::CloudFormation::WaitConditionHandle", "Properties": {}},
+        },
+        "Outputs": {},
+    }
+    output_count = 0
```
Comment (daxAKAhackerman) [inline exporter/lambda/cross_region_import_replication.py:108]: Does this variable have a purpose?

**Review by daxAKAhackerman — CHANGES_REQUESTED** [review]
LGTM appart for my 2 comments. However, I don't think I understand enough how this whole thing is working to be a worthy approver! I think you'll have to explain it to me (again) one when we'll have time!

**Review by daxAKAhackerman — APPROVED** [review]

File: exporter/lambda/cross_region_import_replication.py [hunk 3]
```diff
@@ -100,63 +81,46 @@ def _generate_hash(string_to_hash):
...
 
-def _generate_nested_template(cross_stack_references, max_group_size):
-    last_ref_id = None
-    ssm_resources = {}
-    resource_count = 0
+def _generate_nested_template(cross_stack_references):
+    template = {
+        "AWSTemplateFormatVersion": "2010-09-09",
+        "Resources": {
+            "PlaceHolderParameter": {"Type": "AWS::CloudFormation::WaitConditionHandle", "Properties": {}},
```
Comment (laurrentt) [inline exporter/lambda/cross_region_import_replication.py:104]:
```suggestion
            "PlaceHolderResource": {"Type": "AWS::CloudFormation::WaitConditionHandle", "Properties": {}},
```

File: exporter/lambda/cross_region_import_replication.py [hunk 4]
```diff
@@ -5,92 +5,73 @@
...
+            master_template_resources[f"Chunk{i}"] = {"Type": "AWS::CloudFormation::Stack", "Properties": {"TemplateURL": url}}
     else:
         master_template_resources = {
-            'PlaceHolderParameter': {
-                'Type': 'AWS::SSM::Parameter',
-                'Properties': {
-                    'Value': {'Ref': "AWS::StackName"},
-                    'Type': 'String'
-                },
+            "PlaceHolderParameter": {
```
Comment (laurrentt) [inline exporter/lambda/cross_region_import_replication.py:44]:
```suggestion
            "PlaceHolderResource": {
```

File: exporter/lambda/cross_region_import_replication.py [hunk 5]
```diff
@@ -5,92 +5,73 @@
...
+        cross_stack_references.extend(scan_response["Items"])
 
     if cross_stack_references:
-        number_of_chunk = len(cross_stack_references) / MAX_RESOURCES_PER_TEMPLATE
-        max_group_size = int(max(min(RESSOURCE_BY_GROUP/number_of_chunk, RESSOURCE_BY_GROUP), 1))
-
         nested_template_urls = []
         for items in _chunks(cross_stack_references, MAX_RESOURCES_PER_TEMPLATE):
-            nested_template_urls.append(_generate_nested_template(items, max_group_size))
+            nested_template_urls.append(_generate_nested_template(items))
```
Comment (laurrentt) [inline exporter/lambda/cross_region_import_replication.py:37]: (one line above) rename `MAX_RESOURCES_PER_TEMPLATE` for `MAX_OUTPUTS_PER_TEMPLATE`

**Review by laurrentt — CHANGES_REQUESTED** [review]

**Review by laurrentt — APPROVED** [review]

**Pull request merged**